_tut/docs/3x/execution/cancelable.md

```diff
@@ -294,14 +294,14 @@ def delayedExecution(cb: () => Cancelable)
 }
 ```
 
-## SingleAssignmentCancelable
+## SingleAssignCancelable
 
-The `SingleAssignmentCancelable` is similar to the
-`MultiAssignmentCancelable`, except that it can be assigned once and
+The `SingleAssignCancelable` is similar to the
+`OrderedCancelable`, except that it can be assigned once and
```

Comment:
should it be `OrderedCancelable` or `MultiAssignCancelable` in this sentence?

Reply:
At first I did think of `MultiAssignCancelable`. But the API of [SingleAssignCancelable](https://monix.io/api/3.0/monix/execution/cancelables/SingleAssignCancelable.html) instead used `OrderedCancelable`.  Hence used the same to maintain consistency.

If the API description was unintended, will modify this pull request to use multi  assign instead and create another one to update the scaladoc. The same point is also applicable for `SerialCancelable`.

Reply:
I see, I'm merging it as is then